### DISCUSSION TITLE
Questions about efficientdet D0 network ouput

### DISCUSSION THREAD

**Discussion by minjung727** [rwightman/efficientdet-pytorch#257] [Q&A]
Hi, I am following efficientdet network algorithm, and I have questions about `efficientdet D0` output size.
I got tensor size of [[1, 810, 64, 64], [1, 810, 32, 32], [1, 810, 16, 16], [1, 810, 8, 8], [1, 810, 4, 4]] for `x_class` output, and
[[1, 36, 64, 64], [1, 36, 32, 32], [1, 36, 16, 16], [1, 36, 8, 8], [1, 36, 4, 4]] for `x_box` output.
Can someone tell me the meaning of these outputs? 
I am assuming the last two sizes are about number of grids, but I can't figure out what 810 and 36 means.
Thanks for the help.

**Comment by rwightman** [comment]
@minjung-mmc those outputs need post processing, which is done by the bench wrappers, as is those outputs are relative to the anchors

https://github.com/rwightman/efficientdet-pytorch/blob/master/effdet/bench.py#L20
https://github.com/rwightman/efficientdet-pytorch/blob/master/effdet/anchors.py

You should only use the models without the bench if you plan to write your own post-processing code that's more efficient, for embeded use, higher inference throughput, etc.

**Reply by minjung727** [reply]
Thank you for your answer
